subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1530,11 +1541,18 @@ void ull_adv_done(struct node_rx_event_done *done)
 
 static int init_reset(void)
 {
+	uint8_t handle;
+
 #if defined(CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL) && \
 	!defined(CONFIG_BT_CTLR_ADV_EXT)
 	ll_adv[0].lll.tx_pwr_lvl = RADIO_TXP_DEFAULT;
 #endif /* CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL && !CONFIG_BT_CTLR_ADV_EXT */
 
+	for (handle = 0U; handle < BT_CTLR_ADV_SET; handle++) {
+		lll_adv_data_init(&ll_adv[handle].lll.adv_data);
```

Comment:
primary channel AD Data to be restricted to 37 bytes max. Add new implementation specific to primary channel PDU allocation.

Reply:
Is this a note or a TODO? Should it be added as a TODO in the code?